pulp_2to3_migration/app/plugin/content.py

```diff
@@ -156,11 +156,9 @@ def __init__(self, migrator):
         If a plugin needs to have more control over the order of content migration, it should
         override this method.
         """
-
-        content_types = self.migrator.content_models.keys()
-        for ctype in content_types:
+        for ctype, cmodel in self.migrator.content_models.items():
             # we need to go through all content in case any of Remotes changed
-            pulp2content_qs = Pulp2Content.objects.filter(pulp2_content_type_id=ctype)
+            pulp2content_qs = cmodel.objects.all().prefetch_related('pulp2content')
```

Comment:
We're filtering the QS by type anyway, so we might as well just pass in detail querysets to begin with.

Reply:
@dralley , is it ok for many-to-many relationship as well https://github.com/pulp/pulp-2to3-migration/pull/111/files#diff-7a5490ef73b4f3a627dcbd356eb955a6R108 ?

Reply:
@goosemania  I have read that prefetch_related can prefetch data for many-to-one as well as many-to-many  relationships